sklearn/metrics/pairwise.py

```diff
@@ -1214,6 +1214,7 @@ def pairwise_distances_chunked(X, Y=None, reduce_func=None,
     --------
     Without reduce_func:
 
+    >>> import numpy as np
```

Comment:
please also make sure other required classes/functions are imported. `pairwise_distances_chunked` is an example in this case.